connectors/sources/abs.py

```diff
@@ -85,24 +85,19 @@ def get_default_configuration(cls):
                 "label": "Azure Blob Storage blob endpoint",
                 "type": "str",
             },
-            "connector_name": {
-                "value": "Azure Blob Storage Connector",
-                "label": "Friendly name for the connector",
-                "type": "str",
-            },
             "enable_content_extraction": {
                 "value": DEFAULT_CONTENT_EXTRACTION,
-                "label": "Flag to check if content extraction is enabled or not",
+                "label": "Enable content extraction",
```

Comment:
This is entered as boolean. Can we add `(true/false)` to the name as we did for the other booleans?